aptos-move/framework/aptos-framework/sources/coin.spec.move

```diff
@@ -9,11 +16,169 @@ spec aptos_framework::coin {
 
     spec coin_address {
         pragma opaque;
+        pragma aborts_if_is_partial;
+        aborts_if [abstract] false;
         ensures [abstract] result == spec_coin_address<CoinType>();
     }
 
     spec fun spec_coin_address<CoinType>(): address {
         // TODO: abstracted due to the lack of support for `type_info` in Prover.
         @0x0
     }
-}
+
+    spec initialize_supply_config {
+        let aptos_addr = signer::address_of(aptos_framework);
+        aborts_if !system_addresses::is_aptos_framework_address(aptos_addr);
+        aborts_if exists<SupplyConfig>(aptos_addr);
```

Comment:
nit pick:   these two properties are also guaranteed after execution: 
       ensures !global<SupplyConfig>(aptos_addr).allow_upgrades;
       ensures exists<SupplyConfig>(aptos_addr);

Reply:
done, added the two ensures as suggested.